### DISCUSSION TITLE
Minor New YAML Issue Template Implementation Regression

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#10406] [General]
@Gusted

I have found one unintentional side effect of implementing the new issue templates.

The Security Vulnerability section/links of the Choose your New Issue Template page [here](https://github.com/darkreader/darkreader/issues/new/choose) are duplicated except for minor verbiage differences. See screenshot below.

The cause of the duplication is two things.

1. I believe the Dark Reader repository has a setting for this section to be generated.

2. The new config.yml file that was implemented also has a section in it to generate this as well.

I believe we either need to turn off the repository setting or I can submit another PR to kill this functionality in the config.yml file.

What solution would you prefer to do?

![duplicate](https://user-images.githubusercontent.com/95879668/204396282-b6e0bfad-c8ef-457e-bd71-763d5638775a.png)

**Comment by Gusted** [comment]
> I believe the Dark Reader repository has a setting for this section to be generated.

I don't have permissions to check that... 

> I believe we either need to turn off the repository setting or I can submit another PR to kill this functionality in the config.yml file.

Lets remove it for now in the `config.yml` file.

**Comment by ghost** [comment]
@Gusted

I have created PR #10412 to fix this issue.

I have tested this on my own DR Fork and it worked fine.
